Content.Client/GameObjects/Components/Doors/AirlockVisualizer2D.cs

```diff
@@ -57,6 +59,18 @@ public override void LoadData(YamlMappingNode node)
                 OpenAnimation.AnimationTracks.Add(sound);
                 sound.KeyFrames.Add(new AnimationTrackPlaySound.KeyFrame(openSound, 0));
             }
+
+            DenyAnimation = new Animation {Length = TimeSpan.FromSeconds(0.45f)};
+            {
+                var flick = new AnimationTrackSpriteFlick();
+                DenyAnimation.AnimationTracks.Add(flick);
+                flick.LayerKey = DoorVisualLayers.Base;
+                flick.KeyFrames.Add(new AnimationTrackSpriteFlick.KeyFrame("deny", 0f));
```

Comment:
This doesn't work, the sprite doesn't change, and I can't understand why. The sound plays, though.

Reply:
The animation WAS playing, but the BaseUnlit layer was drawn on top, so now it's made invisible while the animation is playing.